recipes/yourdfpy/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "yourdfpy" %}
+{% set version = "0.0.53" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/yourdfpy-{{ version }}.tar.gz
+  sha256: a026e8b063b9bcb6a251d2b6d2e23edd4de25b98c3ad7699d4d887f2572d3aa0
+
+build:
+  skip: true  # [py<37]
```

This is pure Python and could be noarch:
```suggestion
  noarch: python
```